src/Strategy.sol

```diff
@@ -964,7 +975,7 @@ contract Strategy is BaseHealthCheck, IUniswapV3SwapCallback, AuctionSwapper {
      *  @return Conversion rate
      */
     function _getAssetPerWeth() internal view returns (uint256) {
-        uint256 _answer = (ONE_WAD**2) /
+        uint256 _answer = (ONE_WAD**2) / // review: why **2?
```

Comment:
we are lossing 1e18 when dividing, so to end with 1e18 we need the numerator to be 1e36.